x-pack/plugins/apm/server/lib/transactions/charts/get_timeseries_data/transform.ts

```diff
@@ -36,26 +42,43 @@ export function timeseriesTransformer({
   };
 }
 
-export function getTpmBuckets(
-  transactionResultBuckets: Required<
-    ESResponse
-  >['aggregations']['transaction_results']['buckets'] = [],
-  bucketSize: number
-) {
+type TransactionResultBuckets = Required<
+  ESResponse
+>['aggregations']['transaction_results']['buckets'];
+
+export function getTpmBuckets({
+  transactionResultBuckets = [],
+  bucketSize,
+  durationAsMinutes,
+}: {
+  transactionResultBuckets: TransactionResultBuckets;
+  bucketSize: number;
+  durationAsMinutes: number;
+}) {
   const buckets = transactionResultBuckets.map(
     ({ key: resultKey, timeseries }) => {
       const dataPoints = timeseries.buckets.map((bucket) => {
+        // calculate request/minute. Avoid up-scaling numbers if bucketSize is below 60s (1 minute).
+        // Eg. 1 request during a 10 second window should be displayed as "1 rpm" instead of "6 rpm".
```

Comment:
>         // Eg. 1 request during a 10 second window should be displayed as "1 rpm" instead of "6 rpm".

Is this still the behaviour we want? I thought we were going to display that as 0.1rps

Reply:
@sqren wrong PR 😀 https://github.com/elastic/kibana/pull/80743